local-volume/provisioner/cmd/main.go

```diff
@@ -68,9 +68,17 @@ func main() {
 	node := getNode(client, nodeName)
 
 	glog.Info("Starting controller\n")
+	labelMap := make(map[string]string)
+	for _, labelName := range provisionerConfig.NodeLabelsForPV {
+		labelVal, ok := node.Labels[labelName]
```

Comment:
Check that node.Labels is not nil

Reply:
go-lang sets ok to false if node.Labels is nil. Generally a nil map acts like an empty map for read operations.

Reply:
cool :-)